SmallBetterLogger/SmallBetterLogger.hpp

```diff
@@ -0,0 +1,179 @@
+#ifndef SMALL_BETTER_LOGGER_H
+#define SMALL_BETTER_LOGGER_H
+
+// Used for writing to output stream
+#include <iostream>
+#include <fstream>
+
+// Used for formatting and creating the output string
+#include <sstream>
+#include <vector>
+#include <regex>
+
+namespace sbl
+{
+	// Logger declaration
+	class Logger;
+	using logger = Logger;
```

Comment:
this is not a comment, but I am really curious what these usings do :))

Reply:
`using x = y;` is the equivalent of  `typedef y x;`. In this case, you can use the following synonymous:
`Logger l;`
`logger l;`